VisualRadio/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="altus.visualradio" >
+
```

Comment:
For Android version 4.4(API level 19) and up, permission is not required to read/write files in application specific directories returned by getExternalFilesDir(). 
However, it was an oversite on my part as the minimum API level for thie project is 14 and as such permission msut be added.